websites/models.py

```diff
@@ -99,11 +100,15 @@ def upload_file_to(self, filename):
     )
     markdown = models.TextField(null=True, blank=True)
     metadata = models.JSONField(null=True, blank=True)
-    hugo_filepath = models.CharField(max_length=2048, null=True, blank=True)
+    content_filepath = models.CharField(max_length=2048, null=True, blank=True)
     file = models.FileField(
         upload_to=upload_file_to, editable=True, null=True, blank=True, max_length=2048
     )
 
+    def calculate_checksum(self) -> str:
+        """ Returns a calculated checksum of the content """
+        return sha256(str(self.markdown).encode("utf-8")).hexdigest()
```

Comment:
Should include `metadata` too,  in case the markdown stays the same but the metadata changes.  For example, a `WebsiteContent` object of type `resource` will not have any markdown, just metadata.  Maybe something like this:

```python
sha256(json.dumps(self.metadata) + str(self.markdown).encode("utf-8")).hexdigest()
```